### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
I created a new branch called mobile_menu 
modified Html file to add  div to hold the mobile menu
add javascript file called main.js to handle the event and add an event listener to the hamburger button, to display the mobile button on click

modify the styles.css file to style the mobile menu

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  const targetedDiv2 = document.querySelector('.menu-hidden');
+  targetedDiv2.classList.remove('show');
+  menuButton.style.display = 'block';
+}
+
+closeButton.addEventListener('click', closePop);
+const anchorTagLinks = document.querySelectorAll('.links');
+anchorTagLinks.forEach((link) => {
+  link.addEventListener('click', closePop);
+}); */
```
Comment (rukundoeric) [inline main.js:50]: - [ ] <b>REQUIRED:</b> Kindly, try to keep your code clean by removing this commented-out codes.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+function display() {
+  const targetedDiv = document.querySelector('.menu-hidden');
+  targetedDiv.classList.add('visible');
+  mobileMenuDiv.style.visibility = 'visible';
+  menuButton.style.display = 'none';
+}
+
+const closeMenu = document.querySelector('.closeMenu');
+menuButton.addEventListener('click', display);
+function closePop() {
```
Comment (rukundoeric) [inline main.js:94]: - [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

File: styles.css [hunk 3]
```diff
@@ -3,15 +3,76 @@
...
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 99;
+
+  /* flex-direction: column;
+  align-items: center;
+   background-color: rgba(0, 0, 0, 0.836); */
+}
```
Comment (rukundoeric) [inline styles.css:28]: - [ ] <b>REQUIRED:</b> Kindly, try to keep your code clean by removing this commented-out codes.

File: styles.css [hunk 4]
```diff
@@ -716,6 +777,7 @@ input {
     width: auto;
   }
```
Comment (rukundoeric) [inline styles.css:779]: - [ ] <b>REQUIRED:</b> There is a gap on the right side when the screen is less than `1100px`.  Kindly fix it.

| <=480px      |  >480px & <1100px |
  | ----------- | ----------- |
| Screen Shot 2021-09-08 at 18 21 29 | Screen Shot 2021-09-08 at 18 40 30 |

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @AlaaAlsalem, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.js (hunk 1 above)
Comment (Davidosky007) [reply main.js]: * resloved

File: styles.css (hunk 4 above)
Comment (Davidosky007) [reply styles.css]: * This still needs adjustment kindly fix the white on the left by making the width of the background bigger 
* [ORIGINAL]
image
[YOURS]
* image

**Comment by Davidosky007** [comment]
## Hi @AlaaAlsalem 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
* Kindly make the requested changes requested by the previous reviewer https://github.com/AlaaAlsalem/Aladdin_portfolio/pull/8#discussion_r704596078
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 5]
```diff
@@ -1,15 +1,76 @@
```
Comment (safafa) [inline styles.css:1]: - [ ] Kindly make the change requested by the previous reviewer. 
image

- [ ] [OPTIONAL ] The menu is working. But the menu button only worked for the first time I clicked on it when I close the menu or click on the navigation link the menu Icon disappears and only appears after refreshing the page. I took the screenshot after testing the menu icon for the first time. I think fixing this issue will make a better user experience.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @AlaaAlsalem , Good job working on some of the changes requested by the previous reviewer 👏🏼 👏🏼 

But,  there are still some which aren't addressed yet.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

As the white space wasn't affected by the mobile menu and isn't part of this project, this project should be approved, however, I strongly recommend you to fix this issue and also fix the menu button disappearing after the first usage.

Keep rocking 🚀!

**Pull request merged**